java/src/com/ibm/streamsx/rest/primitives/InstancesArray.java

```diff
@@ -0,0 +1,51 @@
+/*
+# Licensed Materials - Property of IBM
+# Copyright IBM Corp. 2017
+ */
+package com.ibm.streamsx.rest.primitives;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.ibm.streamsx.rest.StreamsConnection;
+import com.ibm.streamsx.rest.primitives.Instance;
+import com.ibm.streamsx.rest.primitives.InstanceGson;
+
+public class InstancesArray {
```

Comment:
Can the purpose/role of these `*Array` classes be explained?

Somewhat of an anti-pattern to have to have such a class, would have just expected `List<Instance>`.